src/backend/controllers/userController.js

```diff
@@ -164,7 +164,7 @@ exports.login = async (req: Request, res: Response) => {
 			return res
 				.status(401)
 				.json({ msg: 'The user name ' + userName + 
-					' is not associated with any account. Double-check your user name and try again.' });
+					' is not associated with any account.' });
```

Comment:
This change is fine, but now I'm just generally questioning what messages we give as responses and whether in this case we should give a more generic one (security me musing to myself)

Reply:
Not that this should be changed now, since it's fine as it is, but looking back at it now for this check we should have grouped the check for a user and pw match as a single condition and send a single response back if either failed rather than check them in order.